src/Defra.Trade.ReMoS.AssuranceService.UI.Core/Integration/ApiIntegration.cs

```diff
@@ -378,19 +379,26 @@ public async Task<Guid> UpdateTradePartyAuthRepSelfServeAsync(TradePartyDto trad
     /// </summary>
     /// <param name="tradePartyId"></param>
     /// <returns>List of establishments</returns>
-    public async Task<List<LogisticsLocationDto>?> GetEstablishmentsForTradePartyAsync(Guid tradePartyId, bool isRejected)
+    public async Task<PagedList<LogisticsLocationDto>?> GetEstablishmentsForTradePartyAsync(
+        Guid tradePartyId, 
+        bool includeRejected,
+        string? searchTerm,
+        string? NI_GBFlag,
+        int pageNumber = 1,
+        int pageSize = 50)
```

Comment:
is 50 the right one for this if default is 10?

Reply:
This is to enable cases where pagination is not required, for self-serve dashboard, it is 10. Dont want to cause issues where pagination is not enabled, so just chose a safe number to return all establishments.